src/main/java/org/jabref/logic/importer/ParserResult.java

```diff
@@ -19,6 +19,7 @@
 import org.jabref.model.metadata.MetaData;
 
 public class ParserResult {
+    public boolean wasChangedOnMigration = false;
```

Comment:
Sorry, but there is no way I am going to sign off a public instance variable such as this. Please follow the Java Conventions and use a getter and a setter.

Reply:
Yeah I though somebody might complain I just find it so stupid... The `protection' is just the same with getters and setters...

Reply:
I know a new book that's about to come out that argues for keeping to conventions when programming Java ;-) Maintainability and all.

Reply:
The "protection" is not the same. Getters and setters encapsulate the variable and might perform (in future) addtional operations on them. That's what I liked about the automatic properties of C#.  Automatically generated getters and setters.